torch/csrc/api/include/torch/nn/options/pooling.h

```diff
@@ -144,5 +144,28 @@ using MaxUnpool2dOptions = MaxUnpoolOptions<2>;
 /// `MaxUnpoolOptions` specialized for 3-D maxunpool.
 using MaxUnpool3dOptions = MaxUnpoolOptions<3>;
 
+// ============================================================================
+
+/// Options for a `D`-dimensional lppool functional and module.
+template <size_t D>
+struct LPPoolOptions {
+  LPPoolOptions(ExpandingArray<D> kernel_size)
+      : kernel_size_(kernel_size), stride_(kernel_size) {}
```

Comment:
I believe `norm_type` is a non-optional argument for the `LPPool1d` constructor in Python version, and we should take it in `LPPoolOptions`'s constructor as well.

Reply:
OK. I added `norm_type` as a non-optional argument.